.github/renovate.json5

```diff
@@ -1,34 +1,5 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "github>newrelic/coreint-automation:renovate-base.json5"
   ],
-  "labels": ["dependencies"],
-  "regexManagers": [
-      {
-        // Parse bundle image version from `BASE_IMAGE` ARG in Dockerfile.
-        "fileMatch": [
-          "^Dockerfile$"
```

Comment:
this should be reduntant due to the dockerfile manager, moreover in the issue such dependency is present twice